src/puppetlabs/puppetdb/jdbc.clj

```diff
@@ -27,6 +27,12 @@
    (string? (first q))
    (every? (complement coll?) (rest q))))
 
+(def Valid-results-query
```

Comment:
We generally have been favouring the lower case variety for naming schemas, with the name ending in -schema usually to designate its a schema from other vars.